ui/app/models/gcp/config.js

```diff
@@ -65,8 +69,45 @@ export default class GcpConfig extends Model {
     'identityTokenTtl',
   ];
 
+  get isWifPluginConfigured() {
+    return !!this.identityTokenAudience || !!this.identityTokenTtl || !!this.serviceAccountEmail;
```

Comment:
Only one of these needs to be set for WIF to be configured?

Reply:
yep. the backend does not have strict required fields for wif, so to figure out if wif is configured I've confirmed with the backend that if at least one of these fields are set then we can assume wif is configured.

Reply:
The user can set `serviceAccountEmail` as a stand alone and that means it's wif accessType, but if they set `identityTokenAudience` we required `serviceAccountEmail` (may be a miss on the backend validation). Thus, defaulting to any of these fields being set means it's wif.